src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3337,6 +3337,7 @@ public override void PostVisit(DottedNameNode node)
                 var nameRhs = node.Right.Name;
 
                 nameRhs = GetLogicalNodeNameAndUpdateDisplayNames(leftType, node.Right);
+                var nodeNameStr = nameRhs.ToString();
```

Comment:
>nodeNameStr [](http://example.com/codeflow?start=20&length=11)

nit: can we rename this var to 'nameRhsString' - just to make it very clear in the cases below that it's tightly associated to nameRhs?

Reply:
Done